docs/admin/install/eventing/install-eventing-with-yaml.md

```diff
@@ -25,12 +25,20 @@ To install Knative Eventing:
 
 ## Verify the installation
 
-!!! success "Monitor the Knative components until all of the components show a `STATUS` of `Running` or `Completed`:"
+!!! success
+    Monitor the Knative components until all of the components show a `STATUS` of `Running` or `Completed`. To do so, run the command:
 
-    ```{ .bash .no-copy }
+    ```bash
     kubectl get pods --namespace knative-eventing
```

Comment:
Should we use `-n` here instead of `--namespace`? Might wanna check with someone from eng if it's OK to add that to our guidelines if it's not there already?